common/logisticspipes/utils/gui/InputBar.java

```diff
@@ -23,16 +27,15 @@
 
 	public String input1 = "";
 	public String input2 = "";
-	private boolean isActive;
 	private boolean displaycursor = true;
 	private long oldSystemTime = 0;
-	private int searchWidth;
-	private boolean numberOnly;
-	private Align align;
+	private GuiTextField inputField;
+	// private Align align; // not supported
```

Comment:
Why leave this in if it won't work anyways?

Reply:
Delete and forget Align permanently?

Reply:
Yeah. It'll still be in the commit history.